.gitignore

```diff
@@ -35,6 +35,8 @@ Makefile.in
 #
 # Top level generated files specific to this top level dir
 #
+/contrib/intel_qat/src
+/contrib/intel_qat/tmp
```

Comment:
Now that we're no longer providing a script do we need these ignores?

Reply:
Shoot. No absolutely not.
Will send in a fix tomorrow!